pkg/command/server.go

```diff
@@ -3,29 +3,51 @@ package command
 import (
 	"context"
 	"fmt"
+	"os"
+	"path/filepath"
 	"syscall"
 
-	"github.com/micro/cli"
+	"github.com/micro/cli/v2"
 	"github.com/oklog/run"
+	"github.com/owncloud/ocis-accounts/pkg/config"
 	"github.com/owncloud/ocis-accounts/pkg/micro/grpc"
 )
 
 // Server is the entry point for the server command.
-func Server() cli.Command {
-	return cli.Command{
+func Server(cfg *config.Config) *cli.Command {
+	baseDir, _ := filepath.Abs(filepath.Dir(os.Args[0]))
```

Comment:
PLEASE NEVER do such a stupid default, it should be NEVER in the same dir as the binary.

Reply:
Yeah, not happy about this one. What would you set as a default location that works in different OS? 🤔

Reply:
This depends on the user AND the os. Not sure if a default is that great at all.

Reply:
Actually no default os better, since different implementations might not care about this path.